oauth2client/contrib/appengine.py

```diff
@@ -49,9 +49,19 @@
 # TODO(dhermes): Resolve import issue.
 # This is a temporary fix for a Google internal issue.
 try:
-    from google.appengine.ext import ndb
```

Comment:
Feel free to push back and call me fussy, but I'd rather see this as
```
try:
    from oauth2client.contrib import _appengine_ndb
except ImportError:
    _appengine_ndb = None

if _appengine_ndb is None:
    <assignment-to-None statements>
else:
    <assignment-to-not-None statements>
```
. Just yesterday or the day before I was directed to the formal documentation: ["Imported names should always be considered an implementation detail"](https://www.python.org/dev/peps/pep-0008/#public-and-internal-interfaces) (i.e. not part of the public API of the module in which they are imported).